cmd/apiserver/apiserver.go

```diff
@@ -27,8 +27,6 @@ import (
 	"k8s.io/kubernetes/pkg/util/logs"
 
 	"github.com/kubernetes-incubator/service-catalog/cmd/apiserver/app/server"
-	// install our API groups
-	_ "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
```

Comment:
I'm kind of surprised go, lint or something didn't complain that this was unused. I guess it can't tell for anonymous imports as easily.